### PR TITLE
add CFL profile

### PR DESCRIPTION
**Pull request by derhuerst**

### CODE REVIEW

**Comment by derhuerst** [comment]
This is WIP, I need to adapt the branch to the current code base.

File: p/cfl/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		os: 'iPhone OS 9.3.5'
+	}
+	body.ver = '1.16'
+	body.auth = {aid: 'ALT2vl7LAFDFu2dz'}
+	body.lang = 'de'
+
+	return body
+}
+
+const insaProfile = {
```
Comment (juliuste) [inline p/cfl/index.js]: This should be called `cflProfile`

File: p/cfl/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	locale: 'de-DE',
```
Comment (juliuste) [inline p/cfl/index.js]: Shouldn't this be the luxemburgish locale?

File: p/cfl/index.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	timezone: 'Europe/Berlin',
```
Comment (juliuste) [inline p/cfl/index.js]: Shouldn't this be `Europe/Luxembourg`?

**Pull request merged**